### PR TITLE
Refactored Invite and Manage component

### PR DESCRIPTION
**Pull request by sakshikhachane**
### Brief description of what is fixed or changed
Refactored `Invite` and `Manage` subcomponents of `Setting` component of admin. This was yet to be done as per the [proposed style](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/PROPOSAL.md)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [x] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

File: frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx [hunk 1]
```diff
@@ -3,16 +3,16 @@ import style from "./Invite.module.scss";
 
 export const Invite = () => {
   return (
-    <div className={style.cont}>
+    <div className={style["cont"]}>
       <div className="container">
-        <h1 className={style.h1}>Invite Admins</h1>
-        <div className={style.dash}></div>
+        <h1 className={style["h1"]}>Invite Admins</h1>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx:8]: Hey @sakshikhachane I am not able to see this title on the page. Can you please fix it.

Comment (sakshikhachane) [reply frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx:8]: @Kajol-Kumari I am able to see the title on my end. Can you please share the screenshot?

Comment (Kajol-Kumari) [reply frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx:8]: sure!
Screenshot from 2021-02-25 12-46-07

Comment (sakshikhachane) [reply frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx:8]: I have refactored `Invite` and `Manage` subcomponents of `Setting` component of admin. I think there is some confusion.

Comment (Kajol-Kumari) [reply frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx:8]: Ah, My bad!! I was checking wrong page :woman_facepalming:

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**